portal/src/main/webapp/js/src/OncoKBConnector.js

```diff
@@ -1911,17 +1911,17 @@ $.fn.dataTableExt.oSort['sort-icons-asc'] = function(x, y) {
     }
 
     //Compare cosmicCount
-    if (x.mutation && _.isNumber(x.mutation.cosmicCount)) {
-        if (y.mutation && _.isNumber(y.mutation.cosmicCount)) {
-            return x.mutation.cosmicCount < y.mutation.cosmicCount ? 1 : -1;
-        } else {
-            return -1;
-        }
-    } else {
-        return 1;
-    }
-
-    return -1;
+    // if (x.mutation && _.isNumber(x.mutation.cosmicCount)) {
+    //     if (y.mutation && _.isNumber(y.mutation.cosmicCount)) {
+    //         return x.mutation.cosmicCount < y.mutation.cosmicCount ? 1 : -1;
+    //     } else {
+    //         return -1;
+    //     }
+    // } else {
+    //     return 1;
+    // }
```

Comment:
Can we just remove these lines instead of commenting out? Saves some LOC. We can always revert back to a previous change if necessary